.npmrc

```diff
@@ -1,6 +1,6 @@
 loglevel = warn
 save-exact = true
-engine-strict = true
+engine-strict = false
```

Comment:
Can you please share the idea why this flag of ignoring node version is disabled here?

Reply:
Andrey suggested we skip BigInt tests for Node < 21, so we can come back to Node 18 compat 👍🏼